editor/store/effects.js

```diff
@@ -230,7 +230,14 @@ export default {
 		}
 	},
 	REQUEST_POST_UPDATE_FAILURE( action, store ) {
-		const { post, edits } = action;
+		const { post, edits, error } = action;
+
+		if ( error && 'rest_autosave_no_changes' === error.code ) {
```

Comment:
In what situations will we expect `error` to be empty such that we'd want to check its truthiness? If it's expected, we should have a separate unit test for this circumstance as well.

Reply:
I'm not sure here. While `REQUEST_POST_UPDATE` does include the error in the dispatched action, `REQUEST_POST_UPDATE_FAILURE` didn't use it until this change, and several tests didn't include an error either, so I felt it was safer to include this check until someone who knows things better could say for certain if the error would always be there or not.